paper/design.tex

```diff
@@ -5,39 +5,61 @@
 \subsection{Functionality}\label{subsec:functionality} % todo maybe rename it requirements?
 Before we dive into the architecture and detailed design of the tool, this section sketches what functionality the tool should have, from the viewpoint of a user.
 
-Figure \ref{mockup_gui_config} shows how the tool could look like for the user.
+Figure \ref{mockup_gui_config} provides a sketch of the intended user interface design, showing the \textbf{file editor mode}. 
```

Comment:
do we actually want to keep the mockups in the paper? idk seems a bit much, the real screenshots should be enough, I would think

Reply:
I would actually keep the mockups in the paper. Previously I was also intending on adding figures for how we want the different JSON features to look like in the GUI, but that part I would not put in design and instead only have screenshots of the actual result in the implementation section.
The mockup is an important part of the design and also shows how we achieved what we intended to do in the design section.
I'm not sure what the expected paper size is, but I feel like currently we don't have too much and rather could put in more.

Reply:
Also, the mockups are important for understanding of the text in the design chapter. And I would not refer to figures from the implementation chapter from the design chapter.